lib/pbbuilder/pbbuilder.rb

```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
-
-Pbbuilder = Class.new(begin
```

Comment:
**question:** What's the rational to removing this? How will we use the debugger now?

Reply:
I, personally, don't have much inconvenience with two files with same name. But changing two different places from `BasicObject` to regular `Object` does seem like an inconvenience.

And why we're going with `BasicObject` and not `ProxyObject`?

Reply:
The only difference between ProxyObject (and previously? or is it currently?) AS::BasicObject is that they re-add `raise` into the namespace, so that you at least can raise. Copying that method into the pbbuilder class seems to be easier than to do the switcharoo with various ActiveSupport versions / doing const detection?

Reply:
> And why we're going with BasicObject
Sorry, missed this 🤗  So, `BasicObject` is a Ruby module which is provided natively by the VM. The `AS::BasicObject/ProxyObject` is an ActiveSupport thing. If we use the ActiveSupport thing we need to switch depending on the ActiveSupport version, which looks obtuse - and the only thing we win is the `raise`. IMO it is not worth it.

Reply:
Good call with `BasicObject`. I noticed this as well:

https://github.com/rails/rails/blob/d7bd5b1e19f76a22ff19a3eb03d72646d645ab4f/activesupport/lib/active_support/proxy_object.rb#L13-L18